docs/source/tutorials/odsc_tutorial/drill_down.rst

```diff
@@ -0,0 +1,62 @@
+Drilling Down on Performance Differences
+========================================
+
+Often, we would like to gain an understanding about why one method performs
+better than another on a given problem. In this section, we show another type
+of visualization which can shed some light on this question.
+
+Plotting Learning Curves per Trial
+----------------------------------
+
+A useful step towards understanding performance differences between setups is
+to look at the learning curves of trials. Here is a script for creating such
+plots for the methods compared in the
+`study above <comparison.html#a-comparative-study>`__:
+
+.. literalinclude:: ../../../../benchmarking/nursery/odsc_tutorial/transformer_wikitext2/local/plot_learning_curves.py
+   :caption: transformer_wikitext2/local/plot_learning_curves.py
+   :start-after: # permissions and limitations under the License.
+
+Full details about visualization of results in Syne Tune are given in
+`this tutorial <../visualization/README.html>`__. In a nutshell, this is what
+happens:
+
+* The workflow is similar to comparative plots, but here, each setup
+  occupies a different subfigure, and there is no aggregation over seeds
+  (the seed has to be specified in ``results.plot``).
+* Two of the methods compared are multi-fidelity (ASHA, MOBSTER), which is
+  why additional information has to be passed as ``multi_fidelity_params``.
+  This is because learning curves are plotted differently for single-fidelity,
+  multi-fidelity of early stopping and of pause-and-resume type.
+* With ``plot_params.subplots``, we ask for a two-by-two matrix of subfigures.
+  By default, subfigures are oriented as a single row.
+
+.. |Learning curves transformer_wikitext2| image:: img/odsc-learncurves-local-seed0.png
```

Comment:
I don't realy know how to do that

Reply:
For the slides, I am presenting 3 figures which show pairs. But for hosted tutorials, we really want to avoid having too many figures, they are large files.